lib/inspec/fetcher.rb

```diff
@@ -26,7 +26,7 @@ def with_default_fetcher(target)
     end
   end
 
-  Fetcher = FetcherRegistry.new
+  FetcherSystem = FetcherRegistry.new
```

Comment:
I feel weird calling this FetcherSystem. It seems like it should be FetcherRegistry, but I see why you can't do that here.

Reply:
True... but why not `Registry`? It's already namespaced under `Inspec::Fetcher`.

Reply:
Oh... but it's not under `Inspec::Fetcher`... it should be.